src/HttpPsrClient.php

```diff
@@ -30,9 +31,10 @@ public function sendRequest(RequestInterface $request, array $options = []);
      * @param RequestInterface[] $requests
      * @param array              $options
      *
-     * @return ResponseInterface[]
+     * @return BatchResult
      *
      * @throws Exception
+     * @throws BatchException
```

Comment:
i think it would be useful. but we can also skip the second sentence, or put it into the general method description.